cmd/commands/cmd_payments.go

```diff
@@ -1517,6 +1521,16 @@ var forwardingHistoryCommand = cli.Command{
 			Usage: "skip the peer alias lookup per forwarding " +
 				"event in order to improve performance",
 		},
+		cli.Int64SliceFlag{
+			Name: "incoming_chan_ids",
+			Usage: "the short channel ids of the incoming " +
```

Comment:
This should just say `short channel id of the ...` since each `--incoming_chan_ids` can only specify a single ID. But the flag can be specified multiple times in the same command (which we often add to the description with `... filter events by; can be specified multiple times in the same command`.
Otherwise the user might think they need to comma separate the values.
Same for the flag below.